netlify.toml

```diff
@@ -1,6 +1,6 @@
 [build]
 publish = "dist"
-command = "yarn build"
+command = "yarn build:prod"
```

Comment:
Is there any reason for separate commands here? In Vue applications, `serve` is for dev and `build` is for prod already. If the danger is that you don't want to overwrite the app registry, I think we can update the `cp` command in the `init-appregistry` script.

Reply:
There's wasn't a super good reason, it just seemed "bad" somehow to change the `build` script, but looking at it now I think changing it would be fine. I'll defer this discussion to the one below.